Makefile

```
@@ -158,9 +159,84 @@ verify-crd-schema: $(CRD_SCHEMA_CHECK) ## Verify CRD schemas for breaking change
+		cat $(FAILURES_FILE); \
+		rm -f $(FAILURES_FILE); \
+		exit 1; \
+	else \
+		rm -f $(FAILURES_FILE); \
+	fi
+endef
+
 .PHONY: verify-ci
-verify-ci: generate update staticcheck fmt vet verify-api-deps verify-crd-schema verify-docs-nav ## Run the same checks as the GHA verify workflow.
-	$(MAKE) verify-git-clean
+verify-ci: ## Run the same checks as the GHA verify workflow.
```

checkmake (0.3.2)

[warning] Target body for "verify-ci" exceeds allowed length of 5 lines (10).

(maxbodylength)